src/index.html

```diff
@@ -228,18 +233,35 @@
                 selected_lang.addClass('invisible');
             }
         }
+
         function setTime() {
             var time = moment.utc().format('YYYY-MM-DD HH:mm') + ' GMT'
             $(".time").text(time);
         }
+
         function openTradingPage() {
             window.location.href = 'v<version>/main.html' + suffix;
         }
+
         function openContactUs() {
             var url = 'https://www.binary.com/' + (local_storage.get('i18n') || { value: 'en' }).value + '/contact.html'
             var win = window.open(url, '_blank');
             win.focus();
         }
+
+        function addLangChangeListener() {
+            $('.languages #select_language li').each((i, el) => {
+                $(el).click(() => {
+                    var anchor = $(el).find('a')[0];
+                    var lang = anchor && /lang=\w+/.test(anchor.href) && anchor.href.match(/lang=(\w+)/)[1];
```

Comment:
@4p00rv Regex is very powerful, but could be expensive and definitely harder to understand. It is not necessary to use RegEx in places where we could write easy reading code.